package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lighthouse",
-  "version": "8.6.0",
+  "version": "8.7.0",
```

Comment:
pretty weird, wish I had an alternative idea. At least, how about `9.0.0-alpha.0`?

Reply:
this is fine. i considered it but got scared. but sure.